portfolio/src/main/webapp/css/about.css

```diff
@@ -0,0 +1,59 @@
+.blackBackgroundImage, .longGroveImage, .stanfordImage, .hobbiesImage {
+  position: relative;
+  opacity: 1;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+}
+
+.blackBackgroundImage {
+  background-image: url('/images/blackbackground.jpg');
+  min-height: 100%;
+}
+
+.longGroveImage {
+  background-image: url('/images/longgrove.jpg');
+  min-height: 400px;
+}
+
+.stanfordImage {
+  background-image: url('/images/stanford.jpg');
+  min-height: 400px;
+}
+
+.hobbiesImage {
+  background-image: url('/images/beach.jpg');
+  min-height: 400px;
+}
+
+.section {
+  text-align: center;
+  padding: 50px 80px;
+}
+
+.section-dark {
+  background-color: black;
+  color: white;
+}
+
+.abouttext {
+  position: absolute;
+  top: 50%;
+  width: 100%;
+  text-align: center;
+  color: white;
+  font-size: 27px;
+  letter-spacing: 8px;
+  text-transform: uppercase;
+}
+
+.abouttext .border.trans {
+  background-color: transparent;
+}
+
+@media(max-width:568px) {
```

Comment:
Personal preference, but I like to keep my `@media` tags inside the class we're modifying, so it's easier to track.

Some people like to keep all their responsive code separate, so they can see all the code for a specific screen size together. Imo that requires keeping too much css in my head, but if you like it better than go for it!